src/disco/components/Addon/index.js

```diff
@@ -57,7 +57,6 @@ export class AddonBase extends React.Component {
     hasAddonManager: PropTypes.bool.isRequired,
     heading: PropTypes.string.isRequired,
     i18n: PropTypes.object.isRequired,
-    iconUrl: PropTypes.string,
```

Comment:
You can leave this prop actually, but you have to assign it in `mapStateToProps()` I think:

```js
export function mapStateToProps(state, ownProps) {
  // `ownProps.guid` is already "normalized" with `getGuid()` in the
  // `DiscoPane` container component.
  const installation = state.installations[ownProps.guid] || {};
  const addon = getAddonByGUID(state, ownProps.guid);

  return {
    addon,
    ...addon,
    iconUrl: addon ? addon.icon_url : null, // <----
    ...installation,
    status: installation.status || UNKNOWN,
    clientApp: state.api.clientApp,
    // This is required by the `withInstallHelpers()` HOC, apparently...
    platformFiles: addon ? addon.platformFiles : {},
    userAgentInfo: state.api.userAgentInfo,
  };
}
```